### PR TITLE
Update for myghost.org & mega.myghost.org addition

### PR DESCRIPTION
**Pull request by SecretRecipe**
updates for list.json

### CODE REVIEW

File: list.json [hunk 1]
```diff
@@ -54,5 +54,13 @@
         "fee": "2.5",
         "stakebonus" : "1",
         "public_key": "gcs154p3rm80dg3arc5jak3293mkgk45uhfggej3da"
+    },
+    {
+        "label": "Mega.Myghost.Org",
```
Comment (akshaynexus) [inline list.json:59]: Fix capitalization

**Review by SecretRecipe — COMMENTED** [review]
fixed

File: list.json [hunk 2]
```diff
@@ -54,5 +54,13 @@
         "fee": "2.5",
         "stakebonus" : "1",
         "public_key": "gcs154p3rm80dg3arc5jak3293mkgk45uhfggej3da"
+    },
+    {
+        "label": "Mega.Myghost.org",
+        "description": "Alternative ghost community coldstaking pool",
+        "website": "https://mega.myghost.org",
```
Comment (akshaynexus) [inline list.json:61]: always add / at end of url

**Review by SecretRecipe — COMMENTED** [review]
sorry, NOW fixed :D

**Pull request merged**